newsroom-lib/src/main/java/com/theguardian/newsroom/reporter/Reporter.kt

```diff
@@ -3,22 +3,19 @@ package com.theguardian.newsroom.reporter
 import com.theguardian.newsroom.Newsroom
 import com.theguardian.newsroom.model.Event
 
-
-abstract class Reporter<T>(val sourceName: String): ReporterTasks {
-
+abstract class Reporter(private val sourceName: String) : ReporterTasks {
```

Comment:
How do you feel about renaming this `BaseReporter` and calling the interface `Reporter`?